### PR TITLE
[ALLUXIO-1849] Add the Portuguese version of documentation page 'Configuring-Alluxio-with-OSS.md'

### PR DESCRIPTION
**Pull request by caadecarvalho**

### CODE REVIEW

**Comment by caadecarvalho** [comment]
Jenkins, test this please.

File: docs/pt/Configuring-Alluxio-with-OSS.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+[compilar os binários a partir do código fonte](http://alluxio.org/documentation/master/Building-Alluxio-Master-Branch.html) 
+ou [baixar o binário pré-compilador diretamente](http://alluxio.org/documentation/master/Running-Alluxio-Locally.html).
+
+Então se você não ainda não criou o arquivo de configuração do `template`, faça-o:
+
+{% include Common-Commands/copy-alluxio-env.md %}
+
+Também, para preparar a utilização do `OSS` com o Alluxio, crie um `bucket` ou utilize um já existente. Você também deve 
+tomar nota do diretório que você quer utilizar no `bucket`, ou pela criação de um novo diretório no `bucket` ou por utilizar 
+um já existente. Para intenção deste guia, o nome do `bucket OSS` será chamado de `OSS_BUCKET e o diretório será
```
Comment (aaudiber) [inline docs/pt/Configuring-Alluxio-with-OSS.md]: Missing backtick after `OSS_BUCKET

**Comment by aaudiber** [comment]
@caadecarvalho Thanks for translating this page, I've left one minor comment.

**Comment by caadecarvalho** [comment]
Jenkins, test this please.

**Comment by caadecarvalho** [comment]
Thanks, @aaudiber

**Comment by aaudiber** [comment]
+1 LGTM

**Pull request merged**